### PR TITLE
Katello 3.16 config

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: releases/katello/3.16/settings [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FULLVERSION='3.16.0-rc4'
```
Comment (ekohl) [inline releases/katello/3.16/settings:1]: I think by now we're at `3.16.0`

Comment (ehelms) [reply releases/katello/3.16/settings:1]: We're actually at rc4.1

Comment (pcreech) [reply releases/katello/3.16/settings:1]: Technically, it's 3.16.0-rc4.1 iirc, but this is adding my work from last week,to be built upon by the person doing the next release.

Comment (pcreech) [reply releases/katello/3.16/settings:1]: I'll defer to @ehelms if he wants me to update to rc4.1 here or if he'll handle it after this merge

File: releases/katello/3.16/settings [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+FOREMAN_VERSION='2.1'
+OSES="rhel7 el8"
```
Comment (ehelms) [inline releases/katello/3.16/settings:3]: I think this should be only rhel7 now

Comment (pcreech) [reply releases/katello/3.16/settings:3]: I want to say i tried that, but still got errors on release pipelines.  I'll remove, shoudln't be hard to put back if it dies.

**Pull request merged**